DisplayCAL/RealDisplaySizeMM.py

```diff
@@ -129,8 +128,7 @@ def get_display(display_no=0):
         if argyll_display.endswith(" [PRIMARY]"):
             argyll_display = " ".join(argyll_display.split(" ")[:-1])
         for display in _displays:
-            desc = display.get("description")
-            if desc:
+            if desc := display["description"]:
```

Comment:
cool we have the walrus operator :+1: 
we are officially supporting only Python 3.8+ then.